### PR TITLE
Reword role selection dialog (boo#1045180)

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

**Comment by lnussel** [comment]
beware of http://bugzilla.opensuse.org/show_bug.cgi?id=1046619

File: control/control.openSUSE.xml [hunk 1]
```diff
@@ -509,51 +509,46 @@ is the most appropriate desktop for you.</label></desktop_dialog>
...
-          <label>Server is usefull when computer should not have interaction with user and have only command line.</label>
+          <label>The server selection installs a reduced set of packages and offers a text mode interface.</label>
         </server_description>
         <custom>
           <!-- TRANSLATORS: a label for a system role -->
           <label>Custom</label>
         </custom>
         <custom_description>
-          <label>Custom role runs additional dialog where user can select software that fits his requirements.</label>
+          <label>The custom selection allows to select alternative software patterns customize the installation. Additional software is available in the online repositories.</label>
```
Comment (DimStar77) [inline control/control.openSUSE.xml]: The custom selection allows to select alternative software patterns customize the installation.

=> The custom selection allows selecting alternative software patterns to customize the installation.

**Comment by jreidinger** [comment]
I will merge it and send to leap. Just note that missing part is dialog title, which I have to change in code.

**Comment by jreidinger** [comment]
submit request 507047

**Pull request merged**